charts/tidb-cluster/templates/_helpers.tpl

```diff
@@ -73,6 +87,20 @@ config-file: |-
     {{- if .Values.tidb.config }}
 {{ .Values.tidb.config | indent 2 }}
     {{- end -}}
+    {{- if or .Values.enableTLSServer .Values.enableTLSClient }}
+  [security]
+    {{- end -}}
+    {{- if .Values.enableTLSServer }}
+  cluster-ssl-ca = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+  cluster-ssl-cert = "/var/lib/tidb-tls/tidb.crt"
+  cluster-ssl-key = "/var/lib/tidb-tls/tidb.key"
+    {{- end -}}
+    {{- if .Values.enableTLSClient }}
+  ssl-ca = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+  ssl-cert = "/var/lib/tidb-tls/tidb.crt"
+  ssl-key = "/var/lib/tidb-tls/tidb.key"
```

Comment:
Should we support users to configure their own MySQL certificates?

Reply:
cc @tennix @weekface What do you think?
One case is when a user migrates a MySQL instance which already has TLS enabled but doesn't want to update application certificates. The user must configure exiting certificates into tidb-server.
If we're going to support this, do we need to support users to configure secret with an arbitrary name?

Reply:
I'm thinking about the same thing too, but AFAIK we don't have any plan to support custom certs yet, if we do, custom CA importing would also need to be added, and we need to validate user uploaded certs to ensure they are signed by the uploaded CA, that would need more code changes and seems better to be another PR.

Reply:
agree. but it seems not right to use cluster TLS certs as tidb-server TLS server for MySQL clients, focusing on cluster TLS support only in this PR?

Reply:
Since we did not support the custom certs now, I don't think it is necessary to expose these parameters to the user, which will make the user confused or misconfigured.